### DISCUSSION TITLE
Some suggestions

### DISCUSSION THREAD

**Discussion by Hujiaohuxue** [cotes2020/jekyll-theme-chirpy#1842] [Ideas]
### Description

I sometimes think:why this theme(chirpy) isn't able to set a background.
So I think my Github Pages is so monotonous that nobody visit it
![Screenshot 2024-07-06 182545](https://github.com/cotes2020/jekyll-theme-chirpy/assets/172238931/91940e05-2f34-4f8f-8b96-f9e7d4a7e2ab)

**Comment by huanyushi** [comment]
Of course you can, you just need to add some style in `assets/css/jekyll-theme-chirpy.scss`.

As an example, you can visit my [blog](https://huanyushi.github.io/) where I have changed the background of the sidebar.

**Reply by Hujiaohuxue** [reply]
ヾ(o′▽`o)ノ°°谢了大兄弟，请问一下具体咋实现呢？
（兄弟你哪里人）

**Reply by huanyushi** [reply]
@Hujiaohuxue 抱歉太晚回复你，这部分代码修改请见[这里](https://github.com/huanyushi/huanyushi.github.io/blob/main/assets/css/jekyll-theme-chirpy.scss#L37)。你只需要知道 css 如何对网页进行设计，可以依据你所想要的进行个性化修改。你如果想要修改整个网页的背景，也同样是可以的，如果你不会 css 的话可以在网络上查找它的教程自行学习。

**Comment by bethanybeachbum** [comment]
Where did you make the change for the picture in assets/css/jekyll-theme-chirpy.scss?  I can't find the line of code for this change?  Thank you very much

**Comment by Hujiaohuxue** [comment]
多谢同胞🙏

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
